mitosheet/src/utils/actions.tsx

```diff
@@ -441,6 +441,27 @@ export const createActions = (
             searchTerms: ['export', 'download', 'excel', 'csv'],
             tooltip: "Download dataframes as a .csv or .xlsx file."
         },
+        [ActionEnum.Export_Dropdown]: {
```

Comment:
Having this makes the search for actions pretty confusing. Maybe we don't want this, or at least don't want to include it in the search results. (see other review comment)